src/coreclr/jit/gentree.cpp

```diff
@@ -1840,6 +1842,39 @@ void CallArgs::Remove(CallArg* arg)
     assert(!"Did not find arg to remove in CallArgs::Remove");
 }
 
+#if TARGET_WASM
+void CallArgs::RemoveAfter(CallArg* arg)
```

Comment:
This needs a [standard function header](https://github.com/dotnet/runtimelab/blob/24524863c43f97a42a55466b5de4cee3fd6d22fb/src/coreclr/jit/conventions.txt#L57-L81).

Also, `RemovedWellKnownArg` needs to be called for any removed args.

Reply:
Thanks, added both